service/protocol.hpp

```diff
@@ -44,6 +44,7 @@ namespace raven
 
   inline void from_json(const raven::json::json &json_data, config_load &cfg)
   {
+      //! We are checking if the config key keyword is present, otherwise it's will probably a read only value
```

Comment:
```suggestion
      //! We are checking if the config key keyword is present, otherwise it's probably a read only value
```
or
```suggestion
      //! We are checking if the config key keyword is present, otherwise it will probably be a read only value
```
(first is probably the right one)